ext/intl/dateformat/dateformat_parse.c

```diff
@@ -149,14 +158,53 @@ PHP_FUNCTION(datefmt_parse)
 			RETURN_FALSE;
 		}
 	}
-	internal_parse_to_timestamp( dfo, text_to_parse, text_len, z_parse_pos?&parse_pos:NULL, return_value);
+	internal_parse_to_timestamp( dfo, text_to_parse, text_len, z_parse_pos?&parse_pos:NULL, false, return_value);
 	if(z_parse_pos) {
 		zval_ptr_dtor(z_parse_pos);
 		ZVAL_LONG(z_parse_pos, parse_pos);
 	}
 }
 /* }}} */
 
+PHP_METHOD(IntlDateFormatter, parseToCalendar)
+{
+	char*           text_to_parse = NULL;
+	size_t          text_len =0;
+	zval*         	z_parse_pos = NULL;
+	int32_t		parse_pos = -1;
```

Comment:
CS: Either align them properly or don't bother having any spaces, but this is just weird